RentACar/RentACar/CRUD/CarCRUD.cs

```diff
@@ -55,8 +55,15 @@ public Car Create(Car car)
 
         public Car Get(int id)
         {
-            // Code here
-            return new Car();
+            if (!File.Exists(_jsonFile)) return new Car(); //??????????
```

Comment:
Escenario donde no existe el archivo json

Reply:
Well this have different ways to handle. I will name some:

- Return an exception. I **do not like** this option. You can easily handle this scenario and exceptions are expensive. I name this option only because I see many people make use of it, without really needing them. Please read https://ericlippert.com/2008/09/10/vexing-exceptions/
- Return `null`. You can use `null` on the caller to evaluate whether the car was found or not. When you receive null you can print a message saying "Sorry the car is not in our Agency" or something similar. This is the easiest and good approach for now.
- Create a wrapping class that gives you a "result" information. With the result itself if everything went OK or an error code if something went bad. This is handy & scales but maybe not really needed now in this project.

Reply:
Good! I will take the null returns way at first, and later just for practice i can create the wrapping class!
Thank you for taking the time to explain me!